frontend/src/modules/scribble/components/Canvas.vue

```diff
@@ -95,6 +95,10 @@ export default {
       this.$emit("drawAction", drawAction);
     },
 
+    requestHistory: function() {
+      this.$emit("requestHistory");
```

Comment:
Does this still need to exist? Shouldn't the CanvasPanel handle all the requesthistory stuff?

Reply:
good catch. I was juggling between trying to find out which "mounted" to put it under.

Reply:
originally i was trying to put it under the canvas.vue mounted and yea